### ISSUE TITLE
Getting Nginx Connection Errors

### ISSUE THREAD

**Issue by iamjameel006** [makeplane/plane#3567]
I am trying to deploy Plane on EC2 (t3a.medium) but when start the container i am getting following errors:

<img width="863" alt="image" src="https://github.com/makeplane/plane/assets/117569030/7cab7ee9-2be2-470b-a214-f59017fd80ec">

**Comment by mguptahub** [comment]
@iamjameel006 

Kindly confirm the setup process. 
- Did you followed the Self hosting Docker Compose instructions ?
- Code Branch used

**Comment by iamjameel006** [comment]
> @iamjameel006
> 
> Kindly confirm the setup process.
> 
> * Did you followed the Self hosting Docker Compose instructions ?
> * Code Branch used

Yes, i added my instance ip and port 80 in .env and followed these https://docs.plane.so/docker-compose instructions for cloud server.

**Comment by mguptahub** [comment]
@iamjameel006 

Not sure what could have gone wrong during setup. 

- Are you on Ubuntu 22.04? It should work perfectly on that. 
- can you share your `.env` file
- how are you accessing the application - ip or domain mapped to ip
- are you able to get the login page ? or in case of 1st time setup, do you see the "Not yet ready" page ?

**Comment by iamjameel006** [comment]
> @iamjameel006
> 
> Not sure what could have gone wrong during setup.
> 
> * Are you on Ubuntu 22.04? It should work perfectly on that.
> * can you share your `.env` file
> * how are you accessing the application - http://ip/ or http://domain/
> * are you able to get the login page ? or in case of 1st time setup, do you see the "Not yet ready" page ?

1. Yes, I am using Ubuntu 22.04.3.

2. Following is my env file i made just 4-5 changes:

NGINX_PORT=80
WEB_URL=http://instanceIP
DEBUG=0
NEXT_PUBLIC_DEPLOY_URL=http://localhost/spaces
SENTRY_DSN=""
SENTRY_ENVIRONMENT="production"
GOOGLE_CLIENT_ID=""
GITHUB_CLIENT_ID=""
GITHUB_CLIENT_SECRET=""
DOCKERIZED=1 # deprecated
CORS_ALLOWED_ORIGINS="http://instanceIP"
# DATA STORE SETTINGS
USE_MINIO=0
AWS_REGION="us-east-1"
AWS_ACCESS_KEY_ID="Here writing my aws access key id"
AWS_SECRET_ACCESS_KEY="Here writing my aws secret key"
AWS_S3_ENDPOINT_URL="https://s3.amazonaws.com"
AWS_S3_BUCKET_NAME="my aws bucket name"

3.I am accessing it by http://ip/

4. Yes for the first time I see "instance is not yet ready page" then i used http://ip/god-mode and able to launch my instance and got login page.

**Comment by mguptahub** [comment]
@iamjameel006 

I hope you are able to access the application .

**Comment by akshat5302** [comment]
Closing this as stale: there hasn't been any activity here for a while, and it was reported against a release that is well behind current Plane (Community v1.4.2 / Commercial v3.2.0).

If you still hit this on a current version, please open a new issue with your Plane version, install method and container logs, and link back here. Thanks!
